setup.py

```diff
@@ -54,6 +54,10 @@ def run(self):
         'Programming Language :: Python :: 3.2',
         'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
```

Comment:
The 3.4 has officially reached it's EOL https://www.python.org/downloads/release/python-3410/